### DISCUSSION TITLE
Great work!

### DISCUSSION THREAD

**Discussion by Daryl-EndicottDavies** [LivingFray/HaloCEVR#17] [General]
Thanks for modding this!  Much appreciated.

**Comment by groybe** [comment]
Thanks LivingFray. This is awesome!
It works on linux via proton too.

**Comment by RocketMan2007** [comment]
Yes, major thank you! This mod is a blast, been looking forward to the OG campaign in VR since VR became a thing again.

**Comment by codyherzog** [comment]
Yeah. Kudos! Thanks for the hard work.

**Comment by slipperfish** [comment]
Thanks for all the time and frustration this took. It would have been tempting to get 6dof aiming, complete one playthrough, and lose interest.
